asgraphite.py

```diff
@@ -831,7 +837,7 @@ def run(self):
                     r = client.info('sindex')
                     if (-1 != r):
                         r = r.strip()
-                        indexes = filter(None, r)
+                        indexes = str(filter(None, r))
```

Comment:
Because that makes it compatible with python 3. Lateswt python3 won't allow a filter type to be used by string, list or dict. In python3, it is a 'filter' type declaration. So you need to cast it to str or list, etc, to actually use it as that. Python2 doesn't mind, or care, python3 requires explicit. This py files makes the code compliant with python2 and python3.

Reply:
This makes it python3 compatible. Python2 doesn't care either way, while python3 requires an explicit declaration to either str/list/dict/etc... Otherwise, you cannot iterate through it as it's of 'filter' type.